docker/package/package_generator.py

```diff
@@ -80,7 +80,6 @@
 ubuntu_versions = [
     "bionic",  # 18.04
     "focal",  # 20.04
-    "hirsute",  # 21.04
```

Comment:
I see that there's also some special treatment of `hirsute` in `docker/package/model.py`, wouldn't it be better to remove that too?

Reply:
@rvem I see you've fixed one there, but there's two more similar lines in that file, please remove those too :slightly_smiling_face:

Reply:
Updated